containers/ecr-viewer/.env.sample

```diff
@@ -9,18 +9,27 @@ AZURE_CONTAINER_NAME=ecr-viewer-files
 
 # AWS Connection
 AWS_REGION=us-east-1
-AWS_CUSTOM_ENDPOINT=http://localhost:4566
 ECR_BUCKET_NAME=ecr-viewer-files
 AWS_ACCESS_KEY_ID=123
 AWS_SECRET_ACCESS_KEY=xyz
+# Only for local use
+AWS_CUSTOM_ENDPOINT=http://localhost:4566
+
+# GCP GCS
+GCS_BUCKET_NAME=ecr-viewer-files
+# Only required if not using Application Default Credentials
+GCP_KEY='{ "client_email": "ecr@example.com", "private_key": "1234" }'
+GCP_PROJECT_ID=projectId
+# Only for local use
+GCS_API_ENDPOINT=http://localhost:10001/
```

Comment:
I get GCS and GCP are different, but the slightly different prefixes here feel like a tripping hazard - would it be still correct to maybe just use GCP for all of them?